easybuild/easyconfigs/n/numpy/numpy-1.11.1-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,25 @@
+name = 'numpy'
+version = '1.11.1'
```

Comment:
@verdurin this is the *exact* same version as the `numpy` that is included in the `Python/2.7.12` module itself, so useless?